sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeServiceClient.cs

```diff
@@ -491,6 +496,9 @@ public virtual DataLakeFileSystemClient GetFileSystemClient(string fileSystemNam
         /// <param name="traits">
         /// Specifies trait options for shaping the file systems.
         /// </param>
+        /// <param name="states">
+        /// Specifies state options for shaping the file systems.
```

Comment:
How's states shaping file systems ?

Reply:
This is for if you want to list deleted file systems.  It works exactly like `BlobServiceClient.GetBlobContainers()`